infrastructure/logging/src/main/java/tech/pegasys/teku/infrastructure/logging/EventLogger.java

```diff
@@ -84,6 +84,11 @@ public void syncEvent(final UInt64 nodeSlot, final UInt64 headSlot, final int nu
     info(syncEventLog, Color.WHITE);
   }
 
+  public void syncCompleted() {
+    final String syncCompleteMsg = String.format("Syncing completed");
+    info(syncCompleteMsg, Color.WHITE);
```

Comment:
nit: I think I'd make this green so it stands out from the other lines as it's an important event:
```suggestion
    info(syncCompleteMsg, Color.GREEN);
```

Reply:
we figured it was already pretty obvious with the line of '***' being broken, but easy enough to change colour, we did talk about whether we should :)